lib/view_component/translatable.rb

```diff
@@ -96,5 +96,13 @@ def translate(key = nil, **options)
     def i18n_scope
       self.class.i18n_scope
     end
+
+    def html_safe_translation(translation)
+      if translation.respond_to?(:map)
+        translation.map { |element| element.respond_to?(:html_safe) ? element.html_safe : element } # rubocop:disable Rails/OutputSafety
```

Comment:
Could we simplify this line by:

```suggestion
        translation.map { |element| html_safe_translation(element) }
```

That way we'd keep the two branched in sync.

Reply:
Good callout! Applied :)

Reply:
In fact, simplified it a bit more: the ternary inside the `if` was a bit weird to begin with. Now it's a simple 3-branch if statement.